Formula/afuse.rb

```diff
@@ -29,6 +29,18 @@ def install
     system "make", "install"
   end
 
+  def caveats
+    on_macos do
+      <<~EOS
+        Reasons for disabling this formula can be found here:
+          https://github.com/Homebrew/homebrew-core/pull/64491
```

Comment:
Now that this is in the caveats, perhaps we could just say outright that macFUSE is not open source and therefore cannot be used as a dependency, instead of pointing users elsewhere to read that.

Reply:
So:
```
    disable! date: "2021-04-08", because: "macFUSE is not permitted (closed-source)"
```
?

Reply:
`disable! date: "2021-04-08", because: "macFUSE is closed-source so not permitted as a Homebrew/core dependency"`

how about that?

Reply:
Yep -- though, let's put something similar in `caveats` as well, as the disable reason currently doesn't show up on https://formulae.brew.sh.

Reply:
https://github.com/Homebrew/formulae.brew.sh/pull/496 should fix the disable reason